### DISCUSSION TITLE
Few features missed

### DISCUSSION THREAD

**Discussion by nightznero** [you-apps/ClockYou#315] [Ideas]
### Describe your suggested feature

Requested Features (from most important to **NOT** so important)
1. Sorting - Manually swipping (on screen posted below; arrows should show after pressing alarm,only to that one specific one is pressed)
2. Smaller label icon + adding colors to them & longer displaying names
3. Days (in letters) in default language to fit app,not only in **English**
4. Snooze repeat count - How many times snooze should be repeated after choosed the time of snooze
5. **In Settings** `2` `5` `6` Colors displaying same color(pink/orange) totally not fit the colors in circles
6. Add _YOU Apps_ color at first option. There is purple but it's in the middle and if im not wrong it's not _YOU Apps_ color as previously
7. Sorting colors cuz its messed

### Other details

here I made visualize to that most important Features
![Picsart_24-03-14_07-03-23-521](https://github.com/you-apps/ClockYou/assets/61750641/26572281-3d88-4701-ad43-0f37168d763e)

### Acknowledgements

- [ ] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I will fill out all of the requested information in this form.

**Comment by SuhasDissa** [comment]
> Add YOU Apps color at first option. There is purple but it's in the middle and if im not wrong it's not YOU Apps color as previously

What android version are you using?

> Sorting colors cuz its messed

We're using [catppuccin](https://github.com/catppuccin/catppuccin) color palette in its default order.

> **In Settings** `2` `5` `6` Colors displaying same color(pink/orange) totally not fit the colors in circles.

Dark mode only issue?

> Days (in letters) in default language to fit app,not only in **English**

Fixed in #267

**Reply by nightznero** [reply]
> What android version are you using?

13 

> Dark mode only issue?

Yea,in light mode its works fine

> > Days (in letters) in default language to fit app,not only in **English**

> Fixed in #267

Thanks 👍🏼 (7.1 issue still presents)
